spec/export/index.md

```diff
@@ -100,36 +102,40 @@ Export data from a FHIR server, whether or not it is associated with a patient.
     </tr>
     <tr>
       <td><code>_type</code></td>
       <td><span class="label label-info">optional</span></td>
+      <td><span class="label label-info">optional</span></td>
       <td>string of comma-delimited FHIR resource types</td>
-      <td>The response SHALL only include resources of the specified resource types(s). If this parameter is omitted, the server SHALL return all supported resources within the scope of the client authorization. For Patient- and Group-level requests, the <a href='https://www.hl7.org/fhir/compartmentdefinition-patient.html'>Patient Compartment</a> SHOULD be used as a point of reference for recommended resources to be returned. However, other resources outside of the patient compartment that are helpful in interpreting the patient data (such as Organization and Practitioner) may also be returned. Servers unable to support <code>_type</code> SHOULD return an error and OperationOutcome resource so clients can re-submit a request omitting the <code>_type</code> parameter.<br /><br />
-      Resource references MAY be relative URLs with the format <code>&lt;resource type&gt;/&lt;id&gt;</code>, or absolute URLs with the same structure rooted in the base URL for the server from which the export was performed. References will be resolved by looking for a resource with the specified type and id within the file set.<br /><br />
-      For example  <code>_type=Practitioner</code> could be used to bulk data extract all Practitioner resources from a FHIR endpoint.</td>
+      <td>The response SHALL be filtered to only include resources of the specified resource types(s).<br /><br />
+      If this parameter is omitted, the server SHALL return all supported resources within the scope of the client authorization, though implementations MAY limit the resources returned to specific subsets of FHIR, such as those defined in the <a href="http://www.fhir.org/guides/argonaut/r2/">Argonaut Implementation Guide</a>. For Patient- and Group-level requests, the <a href='https://www.hl7.org/fhir/compartmentdefinition-patient.html'>Patient Compartment</a> SHOULD be used as a point of reference for recommended resources to be returned. However, other resources outside of the patient compartment that are helpful in interpreting the patient data (such as Organization and Practitioner) may also be returned.<br /><br />
+      Servers unable to support <code>_type</code> SHOULD return an error and OperationOutcome resource so clients can re-submit a request omitting the <code>_type</code> parameter.
+      If the client explicitly asks for export of resources that the bulk data server doesn't support, the server SHOULD return details via an OperationOutcome resource in an error response to the request.<br /><br />
+      For example  <code>_type=Observation</code> could be used to bulk data extract all Observation resources from a FHIR endpoint.</td>
```

Comment:
For 'all Observation resources' this will have to be on the system level. Would specify
```suggestion
      For example  <code><base>/$export?_type=Observation</code> could be used to bulk data extract all Observation resources from a FHIR endpoint.</td>
```
to make that explicit

Reply:
Good point - what if we keep it general and make the text something like: "For example  <code>_type=Observation</code> could be used to filter the response to just Observations.</td>